packages/vaadin-virtual-list/src/vaadin-virtual-list.d.ts

```diff
@@ -0,0 +1,70 @@
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+
+export type VirtualListItem = unknown;
```

Comment:
BTW, please create an issue to add generics support to `vaadin-virtual-list` and link it to #204

Reply:
Done https://github.com/vaadin/web-components/issues/405